src/main/java/net/pms/util/CoverArtArchiveUtil.java

```diff
@@ -534,24 +544,26 @@ private String buildMBReleaseQuery(final CoverArtArchiveTagInfo tagInfo, final b
 			added = true;
 		}
 
-		if (StringUtil.hasValue(tagInfo.artistId)) {
-			if (added) {
-				query.append(AND);
-			}
-			query.append("arid:").append(tagInfo.artistId);
-			added = true;
-		} else if (StringUtil.hasValue(tagInfo.artist)) {
-			if (added) {
-				query.append(AND);
-			}
-			query.append("artistname:");
-			if (fuzzy) {
-				query.append(urlEncode(fuzzString(tagInfo.artist)));
-			} else {
-				query.append(urlEncode("\"" + StringUtil.luceneEscape(tagInfo.artist) + "\""));
-			}
-			added = true;
-		}
+		// Release (album) artist is usually the music director of the album. Track (Recording) artist is usually the singer.
+		// Searching release with artist here is likely to return no result
+//		if (StringUtil.hasValue(tagInfo.artistId)) {
+//			if (added) {
```

Comment:
Can these lines just be removed instead of commented out? Maybe the note could remain, to remind people not to reimplement that logic